server/routes/forms/get.ts

```diff
@@ -3,17 +3,20 @@ import type { ParamsDictionary, PathParams, Query } from 'express-serve-static-c
 import { BadRequest, MethodNotAllowed } from 'http-errors'
 
 import type { BaseData, BaseForm } from '../../forms'
+import type { AppLocals } from './index'
 
 /**
  * Extends express’ normal RequestHandler to be aware that forms are added to locals object
  */
 export type FormGetRequestHandler<
   Forms extends Record<string, BaseForm<BaseData>>,
   Params = ParamsDictionary,
-  ResBody = unknown,
-  ReqBody = unknown,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ResBody = any,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ReqBody = any,
```

Comment:
express also uses `any` here and `unknown` is much more annoying to deal with as typescript cannot infer it and you cannot specify it on request handlers.